pkg/v1/exec.go

```diff
@@ -111,24 +126,53 @@ func (et ExecTask) Execute() (ExecResult, error) {
 	cmd.Stdout = stdoutWriters
 	cmd.Stderr = stderrWriters
 
-	startErr := cmd.Start()
-
-	if startErr != nil {
-		return ExecResult{}, startErr
-	}
+	g := errgroup.Group{}
```

Comment:
I don't think you need this Group, just use the context built into the os/exec https://pkg.go.dev/os/exec#CommandContext

> The provided context is used to kill the process (by calling os.Process.Kill) if the context becomes done before the command completes on its own.

Reply:
I didn't think it had one, thanks for pointing that out. I'll take a look and test it out, to see if behaves in the same way.